### PR TITLE
Passed all tests on scheduling app

### PR DESCRIPTION
**Pull request by jason-alouda**

### CODE REVIEW

**Comment by jason-alouda** [comment]
I have a general code practice question. Is it okay to import everything "import java.util.*" instead of importing exactly what I need?

File: walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java [hunk 1]
```diff
@@ -14,10 +14,58 @@
 
 package com.google.sps;
 
-import java.util.Collection;
+//import java.util.Collection;
+//import java.util.HashSet;
+//import java.util.Set;
+//import java.util.Comparator;
+import java.util.*;
```
Comment (classm-sps) [inline walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java]: to answer your question about this...no :)

https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

File: walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java [hunk 2]
```diff
@@ -14,10 +14,58 @@
...
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    
+    // Arraylist of all available times for meeting request
+    ArrayList<TimeRange> available_times = new ArrayList<TimeRange>();
+
+    // Do a set intersection to extract relevant events
+    ArrayList<Event> relevant_events = new ArrayList<Event>();
+    for (Event ev : events){
```
Comment (classm-sps) [inline walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java]: nit: space before {, everywhere

File: walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java [hunk 3]
```diff
@@ -14,10 +14,58 @@
...
+        Set<String> intersection = new HashSet<String>(ev.getAttendees());
+        intersection.retainAll(request.getAttendees());
+        if (intersection.size() > 0){
+            relevant_events.add(ev);
+        }
+    }
+
+    // Sort events by order of their starting time, then by ending time.
```
Comment (classm-sps) [inline walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java]: This comment doesn't seem to match the code

File: walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java [hunk 4]
```diff
@@ -14,10 +14,58 @@
...
+    // Sort events by order of their starting time, then by ending time. 
+    Comparator<Event> cmp = Comparator.comparing(Event::getWhen, TimeRange.ORDER_BY_START);
+    Collections.sort(relevant_events, cmp);
+
+    int curr_time = TimeRange.START_OF_DAY;
+    for (Event ev : relevant_events){
+        //System.out.println("relevant event:");
+        //System.out.println("Start: " + ev.getWhen().start());
+        //System.out.println("End: " + ev.getWhen().end());
+        //System.out.println("Curr time " + curr_time);
```
Comment (classm-sps) [inline walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java]: remove commented out code, here and below

**Pull request merged**